src/snowflake/snowpark/modin/plugin/extensions/base_overrides.py

```diff
@@ -131,21 +131,76 @@ def decorator(base_method: Any):
 # 4. Performs operations on a native pandas Index object that are nontrivial for Snowpark pandas to manage.
 
 
+@register_base_not_implemented()
+def align(
+    self,
+    other,
+    join="outer",
+    axis=None,
+    level=None,
+    copy=None,
+    fill_value=None,
+    method=lib.no_default,
+    limit=lib.no_default,
+    fill_axis=lib.no_default,
+    broadcast_axis=lib.no_default,
+):  # noqa: PR01, RT01, D200
+    pass  # pragma: no cover
+
+
 @register_base_not_implemented()
 def asof(self, where, subset=None):  # noqa: PR01, RT01, D200
     pass  # pragma: no cover
 
 
+@register_base_not_implemented()
+def at_time(self, time, asof=False, axis=None):  # noqa: PR01, RT01, D200
+    pass  # pragma: no cover
+
+
+@register_base_not_implemented()
+def between_time(
+    self: BasePandasDataset,
+    start_time,
+    end_time,
+    inclusive: str | None = None,
+    axis=None,
+):  # noqa: PR01, RT01, D200
+    pass  # pragma: no cover
+
+
 @register_base_not_implemented()
 def bool(self):  # noqa: RT01, D200
     pass  # pragma: no cover
 
 
+@register_base_not_implemented()
+def clip(
+    self, lower=None, upper=None, axis=None, inplace=False, *args, **kwargs
+):  # noqa: PR01, RT01, D200
```

Comment:
why noqa is needed here, is it because lacking of type hint?

Reply:
It's actually for the lack of docstrings. PR01 is for not documenting a parameter, RT01 for not documenting a return type, and D200 for (possibly) not shortening a one-line docstring.